db/migrate/20180925120130_add_missing_ems_id_to_switch.rb

```diff
@@ -0,0 +1,52 @@
+class AddMissingEmsIdToSwitch < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class Host < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+
+    belongs_to :ext_management_system, :foreign_key => "ems_id"
```

Comment:
All relationships in stubs must use fully qualified class names

Reply:
ok, added